forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -363,3 +364,68 @@ def get():
             )
         except BusinessException as err:
             return err.error, err.status_code
+
+
+@cors_preflight("POST,OPTIONS")
+@API.route("/create-application", methods=["POST", "OPTIONS"])
```

Comment:
This is not an acceptable rest endpoint, as `/application/create` is already there, and `/application/create-application` doesn't follow rest standard.
May be make it as `/application/external/create` or something to align with other endpoint pattern we are following